arithmaticOperation.js

```diff
@@ -0,0 +1,23 @@
+/**
+*   Calculate the area of a rectangle.
+*
+*   length: The length of the rectangle.
+*   width: The width of the rectangle.
+*   
+*	Return a number denoting the rectangle's area.
+**/
+function getArea(length, width) {
+    return length*width;
+}
+
+/**
+*   Calculate the perimeter of a rectangle.
+*	
+*	length: The length of the rectangle.
+*   width: The width of the rectangle.
+*   
+*	Return a number denoting the perimeter of a rectangle.
+**/
+function getPerimeter(length, width) {
```

Comment:
change the argument because one can pass length , width as string , undefined anything , the code will break.

this should be :

function getPerimeter(length : number, width : number)